lib/taperole/commands/ansible.rb

```diff
@@ -0,0 +1,60 @@
+module Taperole
+  module Commands
+    class Ansible < Thor
+      include Taperole::AnsibleRunner
+      include Taperole::Helpers::Logging
+
+      class_option :limit,
+                   type: :string,
+                   aliases: :l,
+                   desc: 'Limits ansible runs to hosts matching the given pattern'
+      class_option :task,
+                   type: :string,
+                   aliases: :T,
+                   desc: 'Name of the rake task to execute'
+      class_option :inventory,
+                   aliases: :i,
+                   type: :string,
+                   desc: 'Do actions with the given inventory file'
+      class_option :name,
+                   aliases: :n,
+                   type: :string,
+                   desc: 'The name of the machine to operate on'
+      class_option :port,
+                   aliases: :p,
+                   type: :numeric,
+                   desc: 'The port that the machine is listening on for SSH connections'
+      class_option :limit,
+                   aliases: :l,
+                   type: :string,
+                   desc: 'Limits ansible runs to hosts matching PATTERN'
+      class_option :tags,
+                   aliases: :t,
+                   type: :string,
+                   desc: 'Only run plays and tasks tagged with these values'
```

Comment:
Nah, it's different. `:t` allows you to only run certain ansible roles, whereas `:T` allows you to run a rake task on the server.